docs/source/tutorials/psij.ipynb

```diff
@@ -0,0 +1,547 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview \n",
+    "In PSI/J’s terminology, a [Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job) represents an executable plus a bunch of attributes. Static job attributes such as resource requirements are defined by the [JobSpec](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_spec.JobSpec) at creation. Dynamic job attributes such as the [JobState](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_state.JobState) are modified by [JobExecutors](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) as the [Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job) progresses through its lifecycle.\n",
+    "\n",
+    "A [JobExecutor](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) represents a specific *Resource Manager*, e.g. Slurm, on which the Job is being executed. Available *Resource Managers*: \n",
+    "- Local\n",
+    "- Cobalt\n",
+    "- Flux\n",
+    "- LSF\n",
+    "- PBS\n",
+    "- Slurm\n",
+    "\n",
+    "Generally, when jobs are submitted, they will be queued for a variable period of time, depending on how busy the target machine is. Once the Job is started, its executable is launched and runs to completion.\n",
+    "\n",
+    "In PSI/J, a job is submitted by [JobExecutor.submit(Job)](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) which permanently binds the Job to that executor and submits it to the underlying resource manager."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Basic Usage\n",
+    "Creating a *single* job with PSI/J:\n",
+    "1. Create a [JobExecutor](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) instance.\n",
+    "2. Create a [JobSpec](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_spec.JobSpec) object and populate it with information about your job\n",
+    "3. Create a [Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job) with that [JobSpec](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_spec.JobSpec)\n",
+    "4. Submit the [Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job) instance to the [JobExecutor](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor)\n",
+    "\n",
+    "### Viewing Job Status\n",
+    "To view a jobs' status, you can use `print(str(Job.status))`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from psij import Job, JobExecutor, JobSpec\n",
+    "\n",
+    "environment = \"local\"\n",
```

Comment:
It's probably a bit confusing to name this variable `environment` since it's too close to "environment" as in "environment variable". Perhaps `executor_type` is a better option.

Reply:
This comment has been addressed